vsts-extension-google-play.json

```diff
@@ -2,7 +2,7 @@
     "manifestVersion": 1.0,
     "id": "google-play",
     "name": "Google Play",
-    "version": "2.142.0",
+    "version": "2.156.0",
```

Comment:
Should this be a major bump? Two of the three tasks had major version changes.

Reply:
Yes, I think so.  I'll do that.